src/neo/Ledger/Blockchain.cs

```diff
@@ -27,7 +27,7 @@ public class Import { public IEnumerable<Block> Blocks; }
         public class ImportCompleted { }
         public class FillMemoryPool { public IEnumerable<Transaction> Transactions; }
         public class FillCompleted { }
-        private class ParallelVerified { public Transaction Transaction; public bool ShouldRelay; public RelayResultReason VerifyResult; }
+        public class ParallelVerified { public Transaction Transaction; public bool ShouldRelay; public RelayResultReason VerifyResult; }
```

Comment:
This shouldn't be changed.

Reply:
Is keeping private nessesary? That makes it difficult to test this type of message in UT.

Reply:
It's nessesary.

Reply:
Let's do it @eryeer, let's copy the initialization of actors from Consensus tests. I can help with some details if needed.